vault_program/src/crank_vault_update_state_tracker.rs

```diff
@@ -43,6 +43,14 @@ pub fn process_crank_vault_update_state_tracker(
         VaultOperatorDelegation::try_from_slice_unchecked_mut(&mut vault_operator_delegation_data)?;
     let ncn_epoch = config.get_epoch_from_slot(slot)?;
 
+    let last_full_state_update_slot = vault.last_full_state_update_slot();
+    let last_full_state_update_epoch = config.get_epoch_from_slot(last_full_state_update_slot)?;
+
+    let operator_last_update_slot = vault_operator_delegation.last_update_slot();
+    let operator_last_updated_epoch = config.get_epoch_from_slot(operator_last_update_slot)?;
+
+    let has_been_partially_updated = last_full_state_update_epoch < operator_last_updated_epoch;
```

Comment:
I think this would never be true - vault slot only gets updated in `close_..` after all the cranks are done (where operator slots are set), so you would never have a vault slot that's higher than an operator slot